### ISSUE TITLE
a bug about blankLinesBetweenScopes

### ISSUE THREAD

**Issue by nypisces** [nicklockwood/SwiftFormat#26]
if a comment after a line, SwiftFormat will add one more line before function, so there are two blank lines
```swift
    public var items: [[BaseItem]] = [[]] // default menu items


    public final override func onPerform<T: Item>(action: Action, item: T) {
```

**Comment by nicklockwood** [comment]
Thanks, I'll investigate!

**Comment by nicklockwood** [comment]
Fixed in 0.9.3
